git-filter-repo

```diff
@@ -2915,10 +2915,11 @@ class RepoAnalyze(object):
     stats = RepoAnalyze.gather_data(args)
 
     # Write the reports
-    sys.stdout.write(_("Writing reports to %s...") % decode(reportdir))
+    sys.stdout.write(_("Writing reports to \"%s\"...") % decode(reportdir))
     sys.stdout.flush()
     RepoAnalyze.write_report(reportdir, stats)
-    sys.stdout.write(_("done.\n"))
+    sys.stdout.write(_(" done.\n"))
```

Comment:
I don't understand the point of adding the space.  Your commit message says it's to make the filename ctrl-clickable I think, but if that were the case then you'd need the space before the ellipsis, right?

Reply:
At least with vscode, ellipsis at the end of a path are ok. So I kept it and simply spaced the "done" from the ellipsis.
![image](https://github.com/user-attachments/assets/8097c490-1a75-4a59-b8f4-c0d61069b267)

In the GIF attached to the PR you can see both paths being underlined correctly on hover.
The quotes improved compatibility, so actually the space is not required with them.

Reply:
Great, since the quotes solved the issue and the extraneous space isn't necessary, let's toss it.